doc/release-notes-1.9.rst

```diff
@@ -33,61 +33,512 @@ The following sections provide detailed lists of changes by component.
 Kernel
 ******
 
-* change description
+* kernel: add early init routines for app RAM
+* kernel: add config for app/kernel split
+* kernel: queue
+* kernel: remove gdb_server
+* kernel: make K_.*_INITIALIZER private to kernel
```

Comment:
the text `K_` looks like a link reference to Sphinx (and it gives an error saying k is an unknown reference). Need to instead write this as `K\_`